src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -6109,13 +5908,13 @@ def get_dummies(
                 )
 
         if prefix is None and not is_series:
+            data_types = self.dtypes
             prefix = [
                 col_name
                 for (col_index, col_name) in enumerate(
                     self._modin_frame.data_column_pandas_labels
                 )
-                if self._modin_frame.is_unnamed_series()
```

Comment:
This is removed here, but not checked in the utils - is that ok?

Reply:
yes, so the outer condition already checked 
```
if prefix is None and not is_series:
```
so if it is not series, then it couldn't be unnamed series also

Reply:
@sfc-gh-rdurrani i want to get the improvement into the coming cut. That seems a small question, I think i can follow up with you on this question offline if you still have some question about this change.